pycbc/events/coinc.py

```diff
@@ -1167,11 +1176,16 @@ def _find_coincs(self, results, valid_ifos):
                 continue
             # Find newly added triggers in fixed_ifo
             trigs = results[fixed_ifo]
+            # Calculate mchirp as a vectorized operation
+            mass1s = trigs['mass1']
+            mass2s = trigs['mass2']
+            mchirps = mchirp_from_mass1_mass2(mass1s, mass2s)
```

Comment:
`mass1s` and `mass2s` are not needed past these few lines, so I would do
```suggestion
        mchirps = mchirp_from_mass1_mass2(
            trigs['mass1'], trigs['mass2']
        )
```